scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java

```diff
@@ -372,18 +371,22 @@ protected void buildCellContextualMenu(Menu menu) {
         }
         priorityItem.setSubmenu(priorityMenu);
 
-        MenuItem removeItem = new MenuItem("Remove", SchedulerImages.instance.job_kill_16().getSafeUri().asString());
-        removeItem.addClickHandler(event -> controller.removeJob(ids));
-        removeItem.setEnabled(selFinished);
-
         MenuItem killItem = new MenuItem("Kill", SchedulerImages.instance.scheduler_kill_16().getSafeUri().asString());
-        killItem.addClickHandler(event -> controller.killJob(ids));
+        killItem.addClickHandler(event -> controller.killJobs(ids));
         killItem.setEnabled(selPauseOrRunning);
 
+        MenuItem killAndResubmitItem = new MenuItem("Kill & Re-submit",
```

Comment:
Same remark for 'Re-submit' --> 'Re-Submit' here.

Reply:
@Mykhailenko @jrochas I think that also goes for the Submit Window title